tox.ini

```diff
@@ -22,6 +22,7 @@ deps =
 passenv =
     PYTHON*
     UBUNTU_IMAGE_*
+    USER
```

Comment:
I guess this is not needed anymore, but let's leave it here. I'll remove it when doing some cleanup later.